src/genie/libs/parser/iosxe/show_device_tracking.py

```diff
@@ -0,0 +1,166 @@
+        # L   10.22.66.10                            7081.0535.b60b     Vl230      230   0100  10194mn REACHABLE
+        # L   10.22.28.10                            7081.0535.b60b     Vl238      238   0100  10255mn REACHABLE
+        # L   10.22.24.10                            7081.0535.b60b     Vl236      236   0100  10330mn REACHABLE
+        # L   10.22.20.10                            7081.0535.b60b     Vl234      234   0100  10329mn REACHABLE
+        # L   10.22.16.10                            7081.0535.b60b     Vl232      232   0100  10330mn REACHABLE
+        # L   10.22.12.10                            7081.0535.b60b     Vl228      228   0100  10330mn REACHABLE
+        # L   10.22.8.10                             7081.0535.b60b     Vl226      226   0100  10329mn REACHABLE
+        # L   10.22.4.10                             7081.0535.b60b     Vl224      224   0100  10329mn REACHABLE
+        # L   10.22.0.10                             7081.0535.b60b     Vl222      222   0100  10329mn REACHABLE
+        # L   10.10.68.10                            7081.0535.b60b     Vl243      243   0100  10330mn REACHABLE
+
+        # Binding Table has 10 entries, 0 dynamic (limit 200000)
+        binding_table_capture = re.compile(
+            r"^Binding\s+Table\s+has\s+(?P<binding_table_count>\d+)\s+entries,\s+(?P<dynamic_entry_count>\d+)\s+dynamic\s+\(limit\s+(?P<binding_table_limit>\d+)\)$")
+        # Codes: L - Local, S - Static, ND - Neighbor Discovery, ARP - Address Resolution Protocol, DH4 - IPv4 DHCP, DH6 - IPv6 DHCP, PKT - Other Packet, API - API created
+        codes_capture = re.compile(
+            r"^Codes:\s+L\s+-\s+Local,\s+S\s+-\s+Static,\s+ND\s+-\s+Neighbor\s+Discovery,\s+ARP\s+-\s+Address\s+Resolution\s+Protocol,\s+DH4\s+-\s+IPv4\s+DHCP,\s+DH6\s+-\s+IPv6\s+DHCP,\s+PKT\s+-\s+Other\s+Packet,\s+API\s+-\s+API\s+created$")
+        # Preflevel flags (prlvl):
+        pref_level_flag_codes_capture = re.compile(r"^Preflevel\s+flags\s+\(prlvl\):$")
+        # 0001:MAC and LLA match     0002:Orig trunk            0004:Orig access
+        pref_level_flags_2_capture = re.compile(
+            r"^0001:MAC\s+and\s+LLA\s+match\s+0002:Orig\s+trunk\s+0004:Orig\s+access$")
+        # 0008:Orig trusted trunk    0010:Orig trusted access   0020:DHCP assigned
+        _capture = re.compile(r"^0008:Orig\s+trusted\s+trunk\s+0010:Orig\s+trusted\s+access\s+0020:DHCP\s+assigned$")
+        # 0040:Cga authenticated     0080:Cert authenticated    0100:Statically assigned
+        _capture = re.compile(r"^0040:Cga\s+authenticated\s+0080:Cert\s+authenticated\s+0100:Statically\s+assigned$")
+        #     Network Layer Address                   Link Layer Address Interface  vlan  prlvl age    state     Time left
+        device_info_header_capture = re.compile(
+            r"^\s+Network\s+Layer\s+Address\s+Link\s+Layer\s+Address\s+Interface\s+vlan\s+prlvl\s+age\s+state\s+Time\s+left$")
```

Comment:
You will trim the space for each line, so you can remove the first ```\s+```  like ```r"^Network\s+Layer

Reply:
Updated